src/read/macho/file.rs

```diff
@@ -481,8 +498,15 @@ pub trait MachHeader: Debug + Pod {
     ///
     /// Also checks that the magic field in the file header is a supported format.
     fn parse<'data, R: ReadRef<'data>>(data: R) -> read::Result<&'data Self> {
+        Self::parse_at_offset(data, 0)
+    }
+
+    fn parse_at_offset<'data, R: ReadRef<'data>>(
+        data: R,
+        offset: u64,
+    ) -> read::Result<&'data Self> {
```

Comment:
Since we have the breaking change for `load_commands` anyway, and you can't do much with a Mach-O file without parsing load commands, can you add the `offset` parameter to `parse` instead of adding a new method?

Reply:
Makes sense, will do!